spec/factories/application_qualification.rb

```diff
@@ -46,6 +48,7 @@
       subject { Hesa::Subject.all.sample.name }
       institution_name { Hesa::Institution.all.sample.name }
       grade { Hesa::Grade.all.sample.description }
+      predicted_grade { false }
```

Comment:
Can degrees not have predicted set to true?

Reply:
@benswannack My bad, yes these can be predicted, amended.